examples/cop-dem-glo-30/Copernicus_DSM_COG_10_N53_00_W115_00_DEM/Copernicus_DSM_COG_10_N53_00_W115_00_DEM.json

```diff
@@ -113,6 +124,129 @@
       "roles": [
         "data"
       ]
+    },
+    "src": {
+      "href": "../../tests/data-files/external/PREVIEW/Copernicus_DSM_COG_10_N53_00_W115_00_SRC.kml",
+      "type": "application/vnd.google-earth.kml+xml",
+      "title": "Source mask",
+      "description": "Source Mask",
+      "role": [
+        "data",
```

Comment:
I feel like this shouldn't have both data and data-mask, like it should be one or the other? maybe `mask` is a sufficient name for this  too?

Reply:
I'm finding other examples around that say "data-mask" but agree that "mask" should be sufficient.